### ISSUE TITLE
PermissionError when running `conda create` as a non-root user

### ISSUE THREAD

**Issue by mconigliaro** [conda/conda#2960]
```
$ conda create -n ipykernel_py2 python=2 ipykernel
Fetching package metadata .......
Solving package specifications: ..........

Package plan for installation in environment /home/cfrazier/.conda/envs/ipykernel_py2:

The following NEW packages will be INSTALLED:

    backports:          1.0-py27_0
    backports_abc:      0.4-py27_0
    decorator:          4.0.10-py27_0
    get_terminal_size:  1.0.0-py27_0
    ipykernel:          4.3.1-py27_0
    ipython:            4.2.0-py27_0
    ipython_genutils:   0.1.0-py27_0
    jupyter_client:     4.3.0-py27_0
    jupyter_core:       4.1.0-py27_0
    libsodium:          1.0.10-0       (soft-link)
    openssl:            1.0.2h-1       (soft-link)
    path.py:            8.2.1-py27_0
    pathlib2:           2.1.0-py27_0
    pexpect:            4.0.1-py27_0
    pickleshare:        0.7.2-py27_0
    pip:                8.1.2-py27_0
    ptyprocess:         0.5.1-py27_0
    python:             2.7.12-0
    pyzmq:              15.2.0-py27_1
    readline:           6.2-2          (soft-link)
    setuptools:         23.0.0-py27_0
    simplegeneric:      0.8.1-py27_1
    singledispatch:     3.4.0.3-py27_0
    six:                1.10.0-py27_0
    sqlite:             3.13.0-0       (soft-link)
    ssl_match_hostname: 3.4.0.2-py27_1
    tk:                 8.5.18-0       (soft-link)
    tornado:            4.3-py27_1
    traitlets:          4.2.1-py27_0
    wheel:              0.29.0-py27_0
    zeromq:             4.1.4-0        (soft-link)
    zlib:               1.2.8-3        (soft-link)

Proceed ([y]/n)? y

Linking packages ...
An unexpected error has occurred, please consider sending the                                                                                                     |   0%
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/opt/anaconda3/bin/conda", line 6, in <module>
    sys.exit(main())
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/cli/main.py", line 120, in main
    exit_code = args_func(args, p)
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/cli/main.py", line 130, in args_func
    exit_code = args.func(args, p)
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/cli/main_create.py", line 57, in execute
    install(args, parser, 'create')
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/cli/install.py", line 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/plan.py", line 595, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/instructions.py", line 136, in execute_instructions
    cmd(state, arg)
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/instructions.py", line 79, in LINK_CMD
    link(state['prefix'], dist, lt, index=state['index'], shortcuts=shortcuts)
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/install.py", line 1042, in link
    with Locked(prefix), Locked(pkgs_dir):
  File "/opt/anaconda3/lib/python3.5/site-packages/conda/lock.py", line 60, in __enter__
    os.makedirs(self.lock_path)
  File "/opt/anaconda3/lib/python3.5/os.py", line 241, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/opt/anaconda3/pkgs/.conda_lock-18799'
```

```
$ conda info
Current conda install:

             platform : linux-64
        conda version : 4.1.4
    conda-env version : 2.5.1
  conda-build version : 1.21.2
       python version : 3.5.1.final.0
     requests version : 2.10.0
     root environment : /opt/anaconda3  (read only)
  default environment : /opt/anaconda3
     envs directories : /home/cfrazier/.conda/envs
                        /home/cfrazier/envs
                        /opt/anaconda3/envs
        package cache : /home/cfrazier/.conda/envs/.pkgs
                        /home/cfrazier/envs/.pkgs
                        /opt/anaconda3/pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/linux-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/linux-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
         offline mode : False
    is foreign system : False
```

**Comment by mconigliaro** [comment]
Is this the same as #2778 ?

**Comment by mconigliaro** [comment]
The fix for me was `conda update conda` to get 4.1.5. Then it worked fine.
